static/_redirects

```diff
@@ -189,6 +189,8 @@
 /docs/home/contribute/blog-post/ /docs/contribute/start/ 301
 /docs/home/contribute/write-new-topic/ /docs/contribute/style/write-new-topic/ 301
 
+/docs/rbac/  /docs/reference/access-authn-authz/rbac/ 301
```

Comment:
hi @jimangel . A couple of thoughts:
I guess there is interest in creating a security docs section from the docs root?
This kind of change opens the door to creating aliases/shortcuts to other sections in the docs (not just security).
I would rather see security add a new section off of docs/home.
I agree that, at times, it is hard to track down pages. The page links are unwieldy or the name of the page title does not match the link string.

Reply:
If adding, this, I'd make it a temporary redirect. The target may change (eg: after adding a security section), and it's not a move of existing content.
```
/docs/rbac  /docs/reference/access-authn-authz/rbac/ 307
/docs/rbac/  /docs/reference/access-authn-authz/rbac/ 307
```

I would also prefer a commit message that says this is a special case and explains why (in other words: not establishing a precedent).

Reply:
@kbhawkey, thanks for the feedback! I have seen short URLs used in the past which is nice for remembering / slide-decks. I don't think we have a policy one way or the other - will look more into this.

At the very least, I think this is worth discussing with the sig-docs community to gauge our appetite for this change / precedent. I will add it to the agenda.

As far as a security section, we do have k8s.io/security - which has been overwritten by a localization team for the second time (unrelated problem :) ). I agree about having a consolidated section for security, which I believe the sig-docs security efforts are targeting.

Reply:
👍

Reply:
There’s a `slug:` attribute available in Hugo front matter. For some examples, look at blog posts in `content/en/blog/_posts/`.

Reply:
moved to markdown file itself verses redirect file.